src/pages/workspace/WorkspacesListPage.js

```diff
@@ -194,7 +192,7 @@ function WorkspacesListPage({policies, allPolicyMembers, reimbursementAccount, u
                     accessibilityLabel={translate('workspace.new.newWorkspace')}
                     success
                     text={translate('workspace.new.newWorkspace')}
-                    onPress={() => App.createWorkspaceAndNavigateToIt('', false, '', false, !isSmallScreenWidth)}
+                    onPress={() => App.createWorkspaceWithPolicyDraftAndNavigateToIt()}
```

Comment:
I can't just do 
```
onPress={App.createWorkspaceWithPolicyDraftAndNavigateToIt}
```
because the `onPress` send a button event, and that object is passed and it breaks the calls